docs/src/components/showcase/TaskCard.tsx

```diff
@@ -31,12 +30,9 @@ export default function TaskCard() {
             March 25th
           </Typography>
         </Box>
-        <Box sx={{ my: 'auto' }}>
-          <CodeRounded />
-          <Typography fontSize={18} component="div" fontWeight="semiBold" sx={{ lineHeight: 1.4 }}>
-            Customize every button and chip instance primary color
-          </Typography>
-        </Box>
+        <Typography fontWeight="semiBold" sx={{ my: 'auto' }}>
```

Comment:
`semiBold` continues to work as this token is defined in the branding theme! It's also a bit more consistent, given we're mostly using worded tokens for font-weights instead of numbers (`medium` instead of `500`, etc.)